packages/backend-dynamic-feature-service/package.json

```diff
@@ -1,8 +1,7 @@
 {
-  "name": "@backstage/backend-plugin-manager",
-  "description": "Backstage plugin management backend",
+  "name": "@backstage/backend-dynamic-feature-service",
+  "description": "Backstage dynamic feature service",
   "version": "0.0.5-next.2",
```

Comment:
Could you set the version to 0.0.0 and add a changeset for the initial release with a minor bump?

Reply:
Done.